readme.md

```diff
@@ -54,7 +54,7 @@ import Lambo from '../../assets/img/gallardo.svg';
 })
 export class MyComponent {
   render() {
-    return <div class='svg-container' innerHTML={Lambo} />;
+    return <img src={Lambo} />;
```

Comment:
While this might make stencil-inline-svg work better in Stencil One, using a <img /> is not an inline svg.

Reply:
Yes, maybe create a parameter to set if it should be inline or not (then maybe the package name becomes obsolete)